lib/ui/semantics.dart

```diff
@@ -636,19 +638,18 @@ class SemanticsUpdateBuilder extends NativeFieldWrapperClass2 {
     Float64List transform,
     Int32List childrenInTraversalOrder,
     Int32List childrenInHitTestOrder,
-    Int32List customAcccessibilityActions,
+    Int32List additionalActions,
   ) native 'SemanticsUpdateBuilder_updateNode';
 
-  /// Update the custom accessibility action associated with the given `id`.
+  /// Update the custom semantics action associated with the given `id`.
   ///
   /// The name of the action exposed to the user is the `label`. The text
   /// direction of this label is the same as the global window.
-  void updateCustomAction({int id, String label}) {
+  void updateCustomAction({int id, String label, String hint, int overrideId = -1}) {
```

Comment:
Update the doc commend above: to include what hint and overrideID is?

Reply:
Done